client/storage-monitor/src/lib.rs

```diff
@@ -42,22 +45,22 @@ pub struct StorageMonitorParams {
 	/// Required available space on database storage. If available space for DB storage drops below
 	/// the given threshold, node will be gracefully terminated. If `0` is given monitoring will be
 	/// disabled.
-	#[arg(long = "db-storage-threshold", value_name = "MB", default_value_t = 1000)]
+	#[arg(long = "db-storage-threshold", value_name = "MiB", default_value_t = 1000)]
 	pub threshold: u64,
 
-	/// How often available space is polled.
-	#[arg(long = "db-storage-polling-period", value_name = "SECONDS", default_value_t = 5, value_parser = clap::value_parser!(u32).range(1..))]
+	/// How often available space is polled in seconds.
```

Comment:
```suggestion
	/// How often available space is polled.
```

Maybe we don't need this?
`--help` is already verbose:
```
      --db-storage-polling-period <SECONDS>
          How often available space is polled
```